api/src/main/java/org/openmrs/module/mambacore/util/MambaETLProperties.java

```diff
@@ -2,126 +2,152 @@
+
+        this.mambaETLuser = properties.getProperty("mambaetl.analysis.db.username",
+                properties.getProperty("connection.username"));
+        this.mambaETLuserPassword = properties.getProperty("mambaetl.analysis.db.password",
+                properties.getProperty("connection.password"));
+
+        this.openmrsDatabase = getProperty(properties, "mambaetl.analysis.db.openmrs_database", "openmrs");
+        this.etlDatababase = getProperty(properties, "mambaetl.analysis.db.etl_database", "analysis_db");
+
+        this.locale = getProperty(properties, "mambaetl.analysis.locale", "en");
+        this.columns = getIntProperty(properties, "mambaetl.analysis.columns", 40);
+        this.incremental = getIntProperty(properties, "mambaetl.analysis.incremental_mode", 1);
+        this.automated = getIntProperty(properties, "mambaetl.analysis.automated_flattening", 0);
+        this.interval = getIntProperty(properties, "mambaetl.analysis.etl_interval", 300);
+        this.tablesWithChangesToStream = Arrays.asList("obs", "encounter");//TODO include more
+    }
+
+    public static synchronized MambaETLProperties getInstance() {
+        if (instance == null) {
+            instance = new MambaETLProperties();
+        }
+        return instance;
+    }
+
+    public String getLocale() {
+        return locale;
+    }
+
+    public int getColumns() {
+        return columns;
+    }
+
+    public int getIncremental() {
+        return incremental;
+    }
+
+    public int getAutomated() {
+        return automated;
+    }
+
+    public int getInterval() {
+        return interval;
+    }
+
+    public String getOpenmrsDbDriver() {
+        return openmrsDbDriver;
+    }
+
+    public String getOpenmrsDbConnectionUrl() {
+        return openmrsDbConnectionUrl;
+    }
+
+    public String getMambaETLuser() {
+        return mambaETLuser;
+    }
+
+    public String getMambaETLuserPassword() {
+        return mambaETLuserPassword;
+    }
+
+    public String getOpenmrsDatabase() {
+        return openmrsDatabase;
+    }
+
+    public String getEtlDatababase() {
+        return etlDatababase;
+    }
+
+    public int getConnectionInitialSize() {
+        return connectionInitialSize;
+    }
+
+    public int getConnectionMaxTotal() {
+        return connectionMaxTotal;
+    }
+
+    public List<String> getTablesWithChangesToStream() {
+        return tablesWithChangesToStream;
+    }
+
+    // Example MambaETLProperties methods
+    public String getSourceDatabaseHost() {
+        return "localhost"; // TODO: put in properties file
+    }
+
+    public String getSourceDatabasePort() {
+        return "3306"; // TODO: put in properties file
+    }
+
+    public String getSourceDatabaseServerId() {
+        return "85744"; // TODO: put in properties file
+    }
+
+    public String getHistoryFilePath() {
+        return "/Users/smallgod/srv/downloads/dbhistory.dat"; // TODO: put in properties file
```

Comment:
This will not work on other people's machines / servers. Please put this on the properties file. In fact everything has this comment

Reply:
Yes, thanks @eudson for your comments.
I am doing to remove all comments and finalise on the cleanups